packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/InputStatus/InputStatus.js

```diff
@@ -1,29 +1,36 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
+import counterpart from 'counterpart'
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
 
 export const StyledInputStatus = styled.span`
   font-size: 0.8em;
   opacity: 0.7;
   text-align: right;
 `
 
-// TODO Add Translations
 export default function InputStatus ({ count, tool }) {
   const minStyleColor = (count < tool.min) ? 'red' : ''
   const maxStyleColor = (count === tool.max) ? 'orange' : ''
+  const drawn = counterpart('InputStatus.drawn')
+  const maximum = counterpart('InputStatus.maximum')
+  const of = counterpart('InputStatus.of')
```

Comment:
I think `of` is a reserved word in javascript (or was it coffeescript)? Perhaps rename this to `ofTranslation`?